APP/src/api.js

```diff
@@ -291,29 +291,41 @@ export const postTeamPictures = async (year, event, team, data, callback) => {
   try {
     const endpoint = `${API_ENDPOINT}/${year}/${event}/frc${team}/pictures/`;
     console.log(endpoint);
-
-    // Convert base64 string to Blob
     const blob = dataURItoBlob(data);
-
-    // Create FormData object
     const formData = new FormData();
-    formData.append('data', blob, 'image.jpg'); // 'data' is the key, 'image.jpg' is the filename
-
+    formData.append('data', blob, 'image.jpg');
```

Comment:
You may want to consider giving the image a different name. Especially if you have additional data about the image. This will help you later if you ever want to build a system to download the image.